examples/mapping_tester_conv/meshes/create_meshes.sh

```diff
@@ -0,0 +1,7 @@
+#!bash
+
+rm -f *.vtk
+
+parallel python ../../../tools/mesh-generators/generate_unit_square.py -r {} -m {}.vtk ::: 0.1 0.0775 0.055 0.0325 0.01 0.06
```

Comment:
Looks like you generate the meshes here. Do you need to push them as well?

Reply:
I included this to more easily change the case in the future. The meshes themselves are pretty small and the generator requires gmsh and meshio. I didn't want to pull these dependencies in the aste CI.